src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -384,6 +384,45 @@ export default {
 				);
 			}
 		},
+		initializeRecommendedLoansRowExp() {
+			// experiment: CASH-1801 "Loans For You" Recommendation Row
+			// get assignment
+			let recommendationChannel;
+			const recommendedLoansRowEXP = this.apollo.readFragment({
+				id: 'Experiment:recommendation_channel',
+				fragment: experimentVersionFragment,
+			}) || {};
+			this.recommendedLoansRowExpVersion = recommendedLoansRowEXP.version;
+			// Only track and activate if these conditions exist
+			if (this.isLoggedIn) {
```

Comment:
@emuvente @mcstover, it looks like the experiment is going to include non-logged in users as well, if I remove `this.isLoggedIn` condition, would we still be able to track how recommendations are working with logged & non-logged in users separately?

Reply:
Yes, if the user is logged in then their user id is sent along with every event that is tracked

Reply:
thanks for the clarification!